### ISSUE TITLE
No overloads for "reversed" on ItemsView

### ISSUE THREAD

**Issue by eddy-geek** [microsoft/pyright#1768]
Note: if you are reporting a wrong signature of a function or a class in the standard library, then the typeshed tracker is better suited for this report: https://github.com/python/typeshed/issues.

**Describe the bug**
False positive 
```python
No overloads for "reversed" match the provided arguments
  Argument types: (ItemsView[Unknown, Unknown])"
````
![image](https://user-images.githubusercontent.com/2772505/115053368-58d23980-9edf-11eb-856f-00e78319bfac.png)

**To Reproduce**
```python
reversed({}.items())
```

**Expected behavior**
It should typecheck

**VS Code extension or command-line**
Pylance v2021.4.1 in vscode 1.55.2 apparently using pyright 1.1.130 patched.

**Comment by erictraut** [comment]
I'm not able to repro the problem with the code snippet you provided. Is there something more needed to repro it?

I noticed in the screen shot above that you also have an "Unexpected indentation" error? Perhaps that's related?

**Comment by erictraut** [comment]
I don't think this is a bug, so I'm going to close it. If you have additional repro steps, feel free to add them, and I'll re-open the issue.
